UI/style.css

```diff
@@ -0,0 +1,505 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", sans-serif;
+}
+
+.container {
+    margin: 0 auto;
+    width: 1200px;
+}
+
+.header{
+    display: flex;
+    align-items: center;
+    height: 77px;
+    background: linear-gradient(245.19deg, #8D46FF 26.86%, #00BAFE 78.69%);
+}
+
+.header__wrapper{
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+}
+
+.header__wrapper_logo{

```

Comment:
From my experience the most commonly used way to write classes and ids in HTML is 'kebab-case': `header-wrapper-logo`.

Reply:
Thank you, I just heard that the development uses the BEM methodology, so the classes have exactly the same names